test/test_linear_fusion_cpu.py

```diff
@@ -0,0 +1,68 @@
+import torch
+import torch.nn as nn
+from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.common_quantization import NodeSpec as ns, QuantizationTestCase
+
+import torchdynamo        
+from torchdynamo.testing import same
+from torchinductor.overrides import fuse_fx, LinearEltwise
+
+torchdynamo.config.raise_on_backend_error = False
+
+def _eltwise_list():
+    eltwise_list = [
+        nn.ReLU(),
+        nn.Sigmoid(),
+        nn.Tanh(),
+        nn.Hardswish(),
+        nn.LeakyReLU(0.1, inplace=False),
+        nn.Hardtanh(min_val=-0.5, max_val=4, inplace=False),
+        nn.GELU(approximate="none"),
+        nn.GELU(approximate="tanh"),
+    ]
+    return eltwise_list
+
+class TestFuseFx(QuantizationTestCase):
```

Comment:
I want leverage this [checkGraphModuleNodes](https://github.com/pytorch/pytorch/blob/ea81138bd6b688554ad307e370bebeffd264f1b7/torch/testing/_internal/common_quantization.py#L583) function to check whether the fusion is successful or not and this function is defined in the [QuantizationTestCase](https://github.com/pytorch/pytorch/blob/ea81138bd6b688554ad307e370bebeffd264f1b7/torch/testing/_internal/common_quantization.py#L332) class.

Reply:
How about we add note to explain why we have to inherit `QuantizationTestCase` here? We may consider to move `checkGraphModuleNodes` util function to a common place later to avoid confusion.

Reply:
Comments are added.